src/main/java/com/cburch/logisim/std/ttl/AbstractOctalFlopsHDLGenerator.java

```diff
@@ -96,11 +97,12 @@
     final var comp = (NetlistComponent) mapInfo;
     var gatedClock = false;
     var hasClock = true;
+    final var cmp = (InstanceFactory) comp.getComponent();
```

Comment:
I see `cmp` I read "compare" :/ But I think that you can get rid of `comp` in line 97 as it looks it is used only once in line 100 or rename former `comp` to i.e. `netlistComp` and refactor `cmp` to be `comp`.

Reply:
fixed